pkg/reconcilers/base_reconciler_test.go

```diff
@@ -21,25 +19,33 @@ package reconcilers
 import (
 	"context"
 	"fmt"
+	"os"
 	"testing"
 
+	"github.com/go-logr/logr"
 	appsv1 "k8s.io/api/apps/v1"
 	v1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/client-go/kubernetes/scheme"
 	"k8s.io/client-go/tools/record"
-	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/client/fake"
 
 	"github.com/kuadrant/limitador-operator/pkg/helpers"
+	"github.com/kuadrant/limitador-operator/pkg/log"
```

Comment:
What about importing the package from Kuadrant?

Reply:
that would be adding adding a dependency to kuadrant-controller, and the kuadrant-controller already has limitador operator as dependency https://github.com/Kuadrant/kuadrant-controller/blob/1d691f6a792cfc4bc0f39de7de3e6c2beba5488c/go.mod#L13

This circular dependency is what I wanted to avoid. 

For me, this "shared" code should be part of some external module as suggested by @didierofrivia . When we have enough shared code to be in a external module, we could create it. Meanwhile, duplicating code is the least problematic issue, IMO.

Reply:
Fair enough.